internal/v6provider/resource_load_balancer_test.go

```diff
@@ -85,6 +86,203 @@ func TestAccKatapultLoadBalancer_basic(t *testing.T) {
 					),
 				),
 			},
+			{
+				ResourceName:      "katapult_load_balancer.main",
+				ImportState:       true,
+				ImportStateVerify: false,
+				ImportStateCheck: func(s []*terraform.InstanceState) error {
+					if len(s) != 1 {
+						return fmt.Errorf(
+							"expected 1 instance state, got %d", len(s),
+						)
+					}
+
+					if s[0].ID == "" {
+						return fmt.Errorf("instance state ID is empty")
+					}
+
+					if s[0].Attributes["name"] != name {
+						return fmt.Errorf(
+							"instance state name is %q, expected %q",
+							s[0].Attributes["name"], name,
+						)
+					}
+
+					return nil
+				},
+			},
+		},
+	})
+}
+
+func TestAccKatapultLoadBalancer_basicWithRule(t *testing.T) {
```

Comment:
We should also have a basic test for creating a load balancer without any rules. Currently trying to do so fails with an error about the internal state of `.rules`, and a message saying it is a bug in the provider.

We should also have a update test that adds rules to a existing LB without any, modifies them, and then also removes all the rules.